pkgs/applications/office/qownnotes/default.nix

```diff
@@ -47,8 +47,11 @@ stdenv.mkDerivation {
     botan2
   ] ++ lib.optionals stdenv.isLinux [ qtwayland ];
 
-  qmakeFlags = [
-    "USE_SYSTEM_BOTAN=1"
+  cmakeFlags = [
+    "-DQON_QT6_BUILD=ON"
+    "-DUSE_SYSTEM_BOTAN=1"
+    "-DBUILD_WITH_SYSTEM_BOTAN=ON"
```

Comment:
Why do we have two options with the same functionality?

Reply:
I'll experiment with the flags a bit.

Reply:
`BUILD_WITH_SYSTEM_BOTAN` alone is ok too